tests/service/connectors/test_queryconfig.py

```diff
@@ -768,10 +804,9 @@ def test_generate_update_stmt_custom_http_method(
         assert prepared_request.path == "/3.0/lists/abc/members/123"
         assert prepared_request.headers == {"Content-Type": "application/json"}
         assert prepared_request.query_params == {}
-        assert (
-            prepared_request.body
-            == '{\n  "merge_fields": {"FNAME": "MASKED", "LNAME": "MASKED"}\n}\n'
-        )
+        assert json.loads(prepared_request.body) == {
+            "merge_fields": {"FNAME": "MASKED", "LNAME": "MASKED"}
```

Comment:
I did this before but now I'm cleaning it up, I didn't like the newlines and spaces in the comparison string.